### ISSUE TITLE
Eeasier theming with theme names

### ISSUE THREAD

**Issue by Disane87** [twbs/bootstrap#29185]
For our customers we want to get the ability to ship some themes based on bootstrap.
Since we don't want to exchange plain css files, it would be great to have a map of themes with the corresponding `map` for the colors you've declared here: https://getbootstrap.com/docs/4.0/getting-started/theming/

Our vision:
We want to ship some different themes and activate them by setting a class to the body. The themes are generated by scss compilation.

So basically an `.theme-normal .alert` would have other colors than `.theme-dark .alert`.

Is that possible?

**Comment by mdo** [comment]
Something like that is possible, but we don't currently support that or plan to.

**Comment by Disane87** [comment]
@mdo any clue of hint how I can solve that?
